src/EditorFeatures/Core/ExternalAccess/VSTypeScript/Api/VSTypeScriptAsynchronousTaggerProvider.cs

```diff
@@ -19,16 +19,27 @@ internal abstract class VSTypeScriptAsynchronousTaggerProvider<TTag> : Asynchron
             IThreadingContext threadingContext,
             IAsynchronousOperationListenerProvider asyncListenerProvider,
             VSTypeScriptGlobalOptions globalOptions)
-            : base(threadingContext, globalOptions.Service, visibilityTracker: null, asyncListenerProvider.GetListener(FeatureAttribute.Classification))
+            : this(threadingContext, globalOptions, visibilityTracker: null, asyncListenerProvider)
         {
         }
 
+        [Obsolete("Use constructor that takes TaggerThreadCoordinator.")]
```

Comment:
@MariaSolOs  you'll get a warning on the next roslyn insertion once this goes in.  Easy to fix.  Just add `TaggerThreadCoordinator threadCoordinator` to your importing tagger constructor